src/common/pure/Modal/index.tsx

```diff
@@ -148,3 +82,62 @@ export default function Modal({
     </>
   )
 }
+
+export const GpModal = styled(Modal)<{
```

Comment:
Since you do the refactor, you could get rid of `GpModal` we are not GP anymore :P

Reply:
What should we call it then?
As "Modal" component is used elsewhere